src/test/functional/steps/email-addresses.ts

```diff
@@ -0,0 +1,90 @@
+import {Then, When} from 'cucumber';
+import {expect} from 'chai';
+
+import * as I from '../utlis/puppeteer.util';
+
+When('I click the Emails tab', async () => {
+  const selector = '#tab_emails';
+  const elementExist = await I.checkElement(selector);
+  expect(elementExist).equal(true);
+  await I.click(selector);
+});
+
+Then('I can view the existing emails', async () => {
+  const elementExist = await I.checkElement('#emailsContent');
+  expect(elementExist).equal(true);
+});
+
+When('I click on Add new Email', async () => {
+  const selector = 'button[name=\'addEmail\']';
+  const elementExist = await I.checkElement(selector);
+  expect(elementExist).equal(true);
+  await I.click(selector);
+});
+
+When('I add Description from the dropdown {string} and Address {string} and Explanation {string} and Welsh Explanation {string}',
+  async (id: string, email: string, explanation: string, explanationCy: string) => {
+
+  const selectSelector = '#newEmailDescription';
+  const addressInputSelector = '#newEmailAddress';
+  const expInputSelector = '#newEmailExplanation';
+  const expCyInputSelector = '#newEmailExplanationCy';
+
+  const selectSelectorExists = await I.checkElement(selectSelector);
+  expect(selectSelectorExists).equal(true);
+  const addressInputSelectorExists = await I.checkElement(addressInputSelector);
+  expect(addressInputSelectorExists).equal(true);
+  const expInputSelectorExists = await I.checkElement(expInputSelector);
+  expect(expInputSelectorExists).equal(true);
+  const expCyInputSelectorExists = await I.checkElement(expCyInputSelector);
+  expect(expCyInputSelectorExists).equal(true);
+
+  await I.selectItem(selectSelector, id);
+  await I.fillField(addressInputSelector, email);
+  await I.fillField(expInputSelector, explanation);
+  await I.fillField(expCyInputSelector, explanationCy);
+});
```

Comment:
Hi @Ayisha-Sharjeel - awesome work on the first commit! Just a heads up, the linter is blocking this from building due to a little indentation here. It spans from lines 28 - 46.

Reply:
Thanks Thomas, I'll fix it.